README.md

```
@@ -345,13 +338,13 @@ A collection of open-source statrter templates, Free templates that are perfect
 </tr>
 </tbody>
 </table>
- </br> </br> 
+ </br> </br>
 
 ## <a name='started'></a> Getting Started with Raaghu
 
 - [Setup Your Development Environment](https://github.com/Wai-Technologies/raaghu-react/wiki#setup-your-development-environment): Start using Raaghu Design System by referring to our docs on Wiki.
 - [Getting Started](https://github.com/Wai-Technologies/raaghu-react/wiki/Getting-Started/#running-the-solution): Create a common Library, Component, Screens etc.
-- [Create new solution](): Guide to create new React Solution. 
+- [Create new solution](): Guide to create new React Solution.
```

markdownlint-cli2 (0.17.2)

No empty links
null

(MD042, no-empty-links)